### PR TITLE
Ticket issues

### LINKED ISSUE #1216: Add ticket number to the order confirmation email
We ran into the problem of people not knowing their ticket number all the time and they often went to check their email. So it'd be really nice to add the ticket number into the email confirmation. And say something like:
Please note: your Ticket Number is XYZ. You need this number to collect your conference badge at the registration desk on the day of the conference.

### PR DESCRIPTION
**Pull request by umgelurgel**
Fixes #1216 #1215 

Ticket configuration page:

image

Order complete page:

image

Sample order complete email:
```
epcon_1  | Hello!
epcon_1  | 
epcon_1  | Thanks for buying tickets for EuroPython 2020.
epcon_1  | 
epcon_1  | This is the summary of order O/20.0008:
epcon_1  | 
epcon_1  | 
epcon_1  | Ticket number: 8 ��� TESP ��� EuroPython 2020 - Early Bird Standard Personal 1/4 ��� 210.00
epcon_1  | 
epcon_1  | Ticket number: 9\ ��� TESP ��� EuroPython 2020 - Early Bird Standard Personal 2/4 ��� 210.00
epcon_1  | 
epcon_1  | Ticket number: 10 ��� TESP ��� EuroPython 2020 - Early Bird Standard Personal 3/4 ��� 210.00
epcon_1  | 
epcon_1  | Ticket number: 11 ��� TESP ��� EuroPython 2020 - Early Bird Standard Personal 4/4 ��� 210.00
epcon_1  | 
epcon_1  | 
epcon_1  | 
epcon_1  | Your tickets were created and are available on your profile page here:
epcon_1  | 
epcon_1  | https://example.com/user-panel/
epcon_1  | 
epcon_1  | Your invoice is also available on the same url.
epcon_1  | 
epcon_1  | If you run into any issues, please contact us at helpdesk@europython.eu
epcon_1  | 
epcon_1  | Kind regards,
epcon_1  | -- 
epcon_1  | EuroPython 2020 Team
epcon_1  | https://ep2020.europython.eu
```

### CODE REVIEW

**Review by aramboi — APPROVED** [review]
Looks good

File: templates/ep19/bs/cart/step_5_congrats_order_complete.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <div class='row'>
         <div class='col-md-12'>
             <h1>Congrats! Order #{{ order.code }} complete.</h1>
-
+            <h4 class="mt-3 mb-3">Please take note of your ticket numbers below.<br/> You will need to provide them during at the registration desk to collect your badge.</h4>
```
Comment (matrixise) [inline templates/ep19/bs/cart/step_5_congrats_order_complete.html:14]: ticket number or ticket numbers?

**Review by matrixise — APPROVED** [review]

**Comment by matrixise** [comment]
Just need to update to the last `dev/ep2020`

File: templates/ep19/bs/cart/step_5_congrats_order_complete.html (hunk 1 above)
Comment (umgelurgel) [reply templates/ep19/bs/cart/step_5_congrats_order_complete.html:14]: Numbers - as per the screenshot above.

**Pull request merged**